recipes/glmnet_py/build.sh

```diff
@@ -0,0 +1,2 @@
+gfortran ./glmnet_python/GLMnet.f -fPIC -fdefault-real-8 -shared -o ./glmnet_python/GLMnet.so
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
Can we use `pip` as in:

https://conda-forge.org/docs/meta.html#use-pip

but in the `build.sh` instead of in the `script` entry? You would need to add `pip` under `host`. Also, does the `setup.py` take care of the `gfortran` compilation step already?

https://github.com/bbalasub1/glmnet_python/blob/master/setup.py#L5-L6

Reply:
I've changed to use `pip`.
For the `gfortran` compilation step: I had found this method of running the `gfortran` command didn't work for some platforms I was testing on so I took it out of the `setup.py` in the tarball here and put it into the `build.sh`. However if you think it would be best to leave it in `setup.py` I can make that change too.

Reply:
You also probably want to use something like `${FC:-gfortran}` instead of gfortran directly